owners/src/types.js

```diff
@@ -63,10 +63,21 @@ let OwnersParserResult;
  */
 let FileRef;
 
+/**
+ * The result of an owners check.
+ *
+ * @typedef {{
+ *   checkRun: CheckRun,
+ *   reviewers: ?string[],
```

Comment:
Why did you choose to have this optional, as opposed to always returning a list, even when it's empty (=> less null checks)

Reply:
Sure, done.